flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/JobStatus.java

```diff
@@ -29,32 +29,35 @@
 	/** Some tasks are scheduled or running, some may be pending, some may be finished. */
 	RUNNING(false),
 
+	/** Source tasks have received STOP signal but are still running (streaming jobs only). */
+	STOPPING(false),
```

Comment:
Is this a legacy artifact from the reverted state machine changes? I cannot find where this state is set in the code.